xikolo-common/Keys/AnalyticsKeys.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  AnalyticsKeys.swift
+//  xikolo-ios
+//
+//  Created by Bjarne Sievers on 18.09.17.
+//  Copyright © 2017 HPI. All rights reserved.
+//
+
+import Foundation
+
+struct AnalyticsKeys {
+    static let visitedDashboard = "VISITED_DASHBOARD"
+    static let visitedAnnouncement = "VISITED_ANNOUNCEMENT"
+    static let visitedItem = "VISITED_ITEM"
+    static let videoEnd = "VIDEO_END"
+    static let videoDownloadStart = "VIDEO_DOWNLOAD_START"
+    static let videoDownloadEnded = "VIDEO_DOWNLOAD_ENDED"
+    static let videoDownloadCanceled = "VIDEO_DOWNLOAD_CANCELED"
+}
```

Comment:
@tobiasrohloff Could you please check this list? It would be great if we use the same set of keys on iOS and Android.